extra/3-magic-8-ball.js

```diff
@@ -56,7 +60,8 @@ function shakeBall() {
     - negative
     - very negative
 
-  This function should expect to be called with any value which was returned by the shakeBall function.
+  This function should expect to be called with any value which was returned 
+  by the shakeBall function.
 */
 function checkAnswer(answer) {
   //Write your code in here
```

Comment:
This function should take any of the possible random answers and then return if the answer is very positive, positive, negative or very negative.